src/pages/Landing/Landing.jsx

```diff
@@ -0,0 +1,73 @@
+import { useNavigate } from "react-router-dom";
+import Button from "components/Button";
+import logo from "assets/logo.png";
+import landing1 from "assets/landing1.png";
+import home1 from "assets/Home-1.png";
+import home2 from "assets/Home-2.png";
+import home3 from "assets/Home-3.png";
+import "./Landing.scss";
+
+export default function Landing({ childeren, onClick }) {
+  const nav = useNavigate();
+
+  return (
+    <div className="view">
+      <div className="CenterLine"></div>
```

Comment:
추가로 첨언하자면 클래스 네임에 현재 사용한 방식은 파스칼 케이스입니다.
우리 팀은 카멜 케이스로 작성하기로 했으니 이렇게 변경할 수 있을 것 같아요

```suggestion
      <div className="centerLine"></div>
```

급한건 아니니 나중에 시간 남으시면 적용해주세요~
참고할만한 게시글 첨부드립니다!

https://ccomccomhan.tistory.com/185#google_vignette